pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -782,20 +832,13 @@ form {
   .flex-container {
     gap: 1.5rem;
     flex-direction: column;
-
-    &.self-paced-english {
-      gap: 1.3rem;
-
-      @media screen and (max-width: $width-md) {
-        gap: 0;
-      }
-    }
```

Comment:
This is a relevant update to this PR. Was able to remove this due to the updated markup.